sys-utils/lsns.8.adoc

```diff
@@ -63,6 +63,12 @@ Do not truncate text in columns.
 *-W*, *--nowrap*::
 Do not use multi-line text in columns.
 
+*-T*, *--nstree* _rel_::
```

Comment:
Cannot we use --tree rather than --nstree?

Reply:
but I see we already use the tree-like output for processes in namespace (lsns <ns>) too. Maybe we can use

 --tree[=parent|owner|process]

and make --tree=process default for `lsns [<ns>]`, and --tree=owner default for `lsns --tree`. What do you think?

Reply:
I think what you wrote is better. Updated my change.